### DISCUSSION TITLE
Updating snap packages: Spot - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#43] [General]
# Updating snap packages: Spot - Alan Pope's blog

I recently lamented that there’s a bunch of broken and outdated snaps in the snap store. Well, some of them are my responsibility, so in the spirit of “be the change you want to see”, let’s get them fixed and updated.
I thought I’d highlight one or two as I go through them, to highlight any important or interesting changes. Today I took a look at Spot, which is a very decent native GNOME Spotify client by Alexandre Trendel.

https://popey.com/blog/2023/10/updated-snap-packages-spot/

**Comment by andypiper** [comment]
The Rick Astley choices for the screenshots are solid, but surely you could have looked up a relevant podcast in the catalog for the images. Linux Matters, perhaps?

**Reply by popey** [reply]
I would, but the app doesn't do podcasts yet... 
![image](https://github.com/popey/popey.com-blog/assets/1841272/97279022-279a-4488-a5b6-9ef8826c51a6)
